.circleci/config.yml

```diff
@@ -73,7 +73,7 @@ workflows:
               only: /^v([0-9])+.([0-9])+.([0-9])+.*/
       - cross-compile:
           requires:
-            - build
+            - setup-and-lint
```

Comment:
I'm not sure if there was any reason why the `cross-compile` step was waiting for the `build` step previously?

I've made the `windows-msi` step run after the `cross-compile` so that it can just make use of the already generated binary & doesn't need to upgrade Go etc. This step can take a couple of minutes and I didn't want to this have to be a fourth step in the pipeline.

i.e.

![image](https://user-images.githubusercontent.com/568630/85813198-ed7b7780-b7a5-11ea-8349-bbff372154e3.png)
vs
![image](https://user-images.githubusercontent.com/568630/85813238-02f0a180-b7a6-11ea-9c8d-355924026c02.png)

Having said that, if we only run this step after merges, maybe this is a moot point.